app.js

```diff
@@ -108,14 +108,22 @@ var q = new ProcessQueue();
 
 function setupWorkDirectory() {
     try {
-        fs.readdirSync(tempPath).forEach(fname => {
-            fs.unlinkSync(tempPath + '/' + fname);
-        });
-        fs.rmdirSync(tempPath);
-    } catch(e) {
-        console.error('work dir does not exist');
+        if (fs.existsSync(tempPath)) {
+            fs.readdirSync(tempPath).forEach(fname => {
+                try {
+                    console.log('Removing file ' + tempPath + '/' + fname);
+                    fs.unlinkSync(tempPath + '/' + fname);
```

Comment:
In the context of Node, it's preferable to use `path.join` instead since it accounts for OS differences and handling slashes.

Reply:
Since this is a fork, I opted to use (minus the `logger` library and ES6 template literals) the change introduced into the parent project, so that if we want to merge from upstream later on it would be easier: https://github.com/fippo/rtcstats-server/blob/master/app.js#L112

I could change this though. Any thoughts?

Reply:
Ok, in that case that's fine.